parser-typechecker/src/Unison/Server/Endpoints/ListNamespace.hs

```diff
@@ -80,16 +80,14 @@ instance ToSample NamespaceListing where
     [ ( "When no value is provided for `namespace`, the root namespace `.` is "
         <> "listed by default"
       , NamespaceListing
-        (Just ".")
-        (Just ".")
+        "."
```

Comment:
I removed the maybe constraint. It seems like that is appropriate, but i'm not sure?

Reply:
We also weren't using both `namespaceListingName` and `namespaceListingFqn`. Removed the former for simplicity.

Reply:
I guess a "name"space listing would have to have a name, not just `Maybe` have a name, so it makes sense to me.